src/source/shift.rs

```diff
@@ -160,6 +161,7 @@ impl SourceParser for Shift {
 
     /// Parse a text source file, using Shift.
     fn parse_file<P: AsRef<Path>>(&self, path: P) -> Result<JSON, Error> {
+        println!("shift");
```

Comment:
Please remove this `println`.

I assume that there are still some `println!` in the code but if you wish to add logging, you should use macros `debug!`, `info!` or `warn!` and set a target, so that we can explicitly enable/disable them.

See https://docs.rs/log/0.4.1/log/ for the documentation on these macros.

Reply:
I have done this for debugging.

Reply:
Sure. Just don't forget to remove them before you submit a patch :)